src/components/cc-tile-metrics/cc-tile-metrics.js

```diff
@@ -0,0 +1,369 @@
+      return 'error';
+    }
+    if (this.metrics.state === 'empty') {
+      return 'empty';
+    }
+    if (this.metrics.state === 'loading' || this.metrics.state === 'loaded') {
+      return 'chart';
+    }
+  }
+
+  _onToggleDocs () {
+    this._docs = !this._docs;
+  }
+
+  firstUpdated () {
+    this._cpuCtx = this.renderRoot.getElementById('cpu_chart');
+    this._memCtx = this.renderRoot.getElementById('mem_chart');
+
+    this._cpuChart = this._createChart(this._cpuCtx);
+    this._memChart = this._createChart(this._memCtx);
+
+  }
+
+  // updated and not update because we need this._chart before
+  updated (changedProperties) {
+
+    if (changedProperties.has('metrics')) {
+
+      this._cpuChart.data = (this.metrics.state === 'loaded')
+        ? this._getChartData(this.metrics.value.cpuData)
+        : this._getChartData(null);
+
+      this._cpuChart.update();
+      this._cpuChart.resize();
+
+      this._memChart.data = (this.metrics.state === 'loaded')
+        ? this._getChartData(this.metrics.value.memData)
+        : this._getChartData(null);
+
+      this._memChart.update();
+      this._memChart.resize();
+    }
+
+  }
+
+  render () {
+
+    const lastCpuValue = (this.metrics.state === 'loaded') ? this.metrics.value.cpuData[this.metrics.value.cpuData.length - 1].value : 0;
+    const lastMemValue = (this.metrics.state === 'loaded') ? this.metrics.value.memData[this.metrics.value.memData.length - 1].value : 0;
+
+    const cpuColorType = (this.metrics.state === 'loaded') ? this._getColorLegend(lastCpuValue) : '#bbb';
+    const memColorType = (this.metrics.state === 'loaded') ? this._getColorLegend(lastMemValue) : '#bbb';
+
+    const panel = this._getCurrentPanel();
+
+    return html`
+      <div class="tile_title">
+        ${i18n('cc-tile-metrics.title')}
+        <div class="docs-buttons">
+          ${ccLink(this.grafanaLink, html`
+            <img class="grafana-logo" src=${grafanaSvg} alt="">
+          `, false, i18n('cc-tile-metrics.link-to-grafana'))}
+          <cc-button
+            class="docs-toggle"
+            image=${(this._docs ? closeSvg : infoSvg)}
+            hide-text
+            @cc-button:click=${this._onToggleDocs}
+          >${this._docs ? i18n('cc-tile-metrics.close-btn') : i18n('cc-tile-metrics.about-btn')}
+          </cc-button>
+        </div>
+      </div>
+      <div class="tile_body ${classMap({ 'tile--hidden': panel !== 'chart' })}">
+        <div class="category">
+          <div>${i18n('cc-tile-metrics.cpu')}</div>
+          <div class="chart-wrapper">
+            <div class="chart-container ${classMap({ skeleton: this.metrics.state === 'loading' })}">
+              <canvas id="cpu_chart"></canvas>
+            </div>
+          </div>
+          <div class="current-percentage ${classMap({ skeleton: this.metrics.state === 'loading' })}" style=${`color: ${cpuColorType}`}>${i18n('cc-tile-metrics.percent', { percent: lastCpuValue / 100 })}</div>
```

Comment:
I was wondering why:
```
style=${`color: ${cpuColorType}`}
```
and not:
```
style="color: ${cpuColorType}"
```
but I'm not sure which one is the best.

Same for the other chart.